org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/SynapseLanguageService.java

```diff
@@ -304,6 +304,12 @@ public void updateConnectors() {
         SchemaGenerate.generate(connectorHolder, connectorPath);
     }
 
+    public void updateInboundConnectors() {
+
+        inboundConnectorHolder.getCustomInboundConnectors(new File(Path.of(projectUri, Constant.SRC, Constant.MAIN, Constant.WSO2MI,
+                Constant.RESOURCES, Constant.INBOUND_CONNECTORS_DIR).toString()), projectServerVersion);
```

Comment:
We already initiated the InboundConnectorHolder with the projectVersion in the `init` method of SynapseLanguageService. We don't need to pass it here.

Reply:
Stored the project runtime version in the InboundConnectorHolder as it is needed to update the inbound connectors. Reference: f2945517fa222e1e1375ddfaa5cb92a45d5605b5